yapapi/runner/__init__.py

```diff
@@ -86,10 +87,8 @@ async def decorate_demand(self, demand: DemandBuilder) -> None:
         self._activity = Activity.from_props(demand.props)
 
     async def score_offer(self, offer: rest.market.OfferProposal) -> float:
-        try:
-            linear: com.ComLinear = com.ComLinear.from_props(offer.props)
-        except Exception:
-            return SCORE_REJECTED
+
+        linear: com.ComLinear = com.ComLinear.from_props(offer.props)
```

Comment:
`com.Linear` would look better. Are you sure it should be `com.ComLinear`?

Reply:
Yep, that's correct, see here: https://github.com/golemfactory/yapapi/blob/ec146fb43d8ebb2e552ce0546e3b6351d86283d5/yapapi/props/com.py#L36